firstvoices/backend/permissions/predicates/base.py

```diff
@@ -1,4 +1,4 @@
-from rules import predicate
+from rules import predicate, Predicate
```

Comment:
This is correcting an error that wasn't causing problems before-- we were creating some predicates using the function decorator (lowercase p) instead of the class (uppercase p). Main effect of that is that using the decorator as a function doesn't name the resulting predicate, so it couldn't get matched up to a filter with the same name. Now these can all be used as view permissions on models without triggering a configuration warning about a missing permission filter